folium/map.py

```diff
@@ -22,7 +22,7 @@
 
 _default_js = [
     ('leaflet',
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"),
+     "https://npmcdn.com/leaflet@1.0.0-rc.3/dist/leaflet.js"),
```

Comment:
That is a release candidate, no? `rc.3`?

Still releases candidates are "stable enough." Let me just run the whole gallery with that version before merging this to be sure nothing is broken.